docs/UserGuide.md

```diff
@@ -162,6 +176,19 @@ Examples:
 * `edit 2 g/f`
 * `edit 3 g/F`
 
+__Optional Parameter 2: Date of Birth__
+
+Format: `edit INDEX [n/NAME] [d/DOB]`
```

Comment:
I think the [d/NAME] may not be necessary here? Especially since we are focusing on editing the DOB only!
```suggestion
Format: `edit INDEX [n/NAME] [d/DOB]`
```